data-preprocessing/JSON  Section/README.md

```diff
@@ -0,0 +1,12 @@
+# How to use
+
+## Useful Information
+The json files are of two forms. The structure of the Algebra.json is slightly different from Chemistry 2e.json and College Physics 2e.json. As such slightly different block of code handle the return of the sections to account for these differences.
```

Comment:
typo. `As such slightly different block of code handle` should either be `As such a slightly different block of code handles` or `As such slightly different blocks of code handle` based on what is actually happening

also will be good to reference the block of code you are referring to here since the guidance is a bit vague